build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -4,6 +4,7 @@
 
   <PropertyGroup>
     <NuGetOutDir>$(OutDir)Packages\</NuGetOutDir>
+    <OutDir>$(OutDir)Raw\</OutDir>
```

Comment:
This is from my original PR.  The output directory was becoming a mess with all of the loc dirs mixed with Packages, PackagesLayouts, ProjectTemplates, Sdks, Tests. At one point, it was interfering with my ability to glob correctly and pick up the satellites, but that may no longer be an issue with other changes I made since.